features/cis/test/test_debian_cis.py

```diff
@@ -1,10 +1,23 @@
-from helper.exception import NotPartOfFeatureError, DisabledBy
-from helper.tests.debian_cis import DebianCIS
 import pytest
+from helper.tests.debian_cis import DebianCIS
+
+
+@pytest.mark.parametrize(
+    "config,args",
+    [
+        ("default", {
+            "git_debian_cis": "https://github.com/ovh/debian-cis.git",
+            "git_debian_cis_branch": "master",
+            "git_gardenlinux": "https://github.com/gardenlinux/gardenlinux.git",
+            "git_gardenlinux_branch": "main",
+            "config_src": "/tmp/gardenlinux/features/cis/test/conf.d/*.cfg",
+            "config_dst": "/tmp/debian-cis/etc/conf.d/",
+            "script_src": "/tmp/gardenlinux/features/cis/test/check_scripts/*.sh",
+            "script_dst": "/tmp/debian-cis/bin/hardening/"
+        })
+    ]
+)
```

Comment:
This parametrize statement could be adjusted. We don't need a list of tuples but could directly pass the dictionary to the function.

So something like this would work:
```
@pytest.mark.parametrize(
    "args",
    [
       {
            "git_debian_cis": "https://github.com/ovh/debian-cis.git",
            "git_debian_cis_branch": "master",
            "git_gardenlinux": "https://github.com/gardenlinux/gardenlinux.git",
            "git_gardenlinux_branch": "main",
            "config_src": "/tmp/gardenlinux/features/cis/test/conf.d/*.cfg",
            "config_dst": "/tmp/debian-cis/etc/conf.d/",
            "script_src": "/tmp/gardenlinux/features/cis/test/check_scripts/*.sh",
            "script_dst": "/tmp/debian-cis/bin/hardening/"
       }
    ]
)
```